test/files/run/repl-paste-parse.check

```diff
@@ -1,5 +1,5 @@
 
-scala>
+scala> 
```

Comment:
It's outspacing me for now. Some interesting design choices around the splash loop, where to get input from, and the loop command.